features/integration/api/authorization.feature

```diff
@@ -196,7 +196,7 @@ Feature: A payment authorization request is made against the Worldpay payment ga
             | numberHighRiskUnits | 0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
-            | shippingAddress     | Tim,Webster,7,Crisswell Close,Crownhill,PA16 0XA,Milton Keynes,Bucks,GB,07426111111                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+            | shippingAddress     | Tim,Webster,7,Crisswell Close,Crownhill,PA16 0XA,Milton Keynes,Bucks,GB,07426111111                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
What's actually changed here? I'm struggling to see the difference between the 2 lines.

Reply:
spacing on the far right of the |